src/scvi/utils/_track.py

```diff
@@ -4,7 +4,7 @@
 
 from rich.console import Console
 from rich.progress import track as track_base
-from tqdm import tqdm as tqdm_base
+from tqdm.auto import tqdm as tqdm_base
```

Comment:
This fixes the progress bar for training?

Reply:
yep